app/controllers/merge_requests_controller.rb

```diff
@@ -5,6 +5,8 @@ class MergeRequestsController < ApplicationController
 
   def create
     @merge_request = MergeRequest.create!(merge_request_params)
+    MergeRequestOrganisation.create!({ merge_request: @merge_request, merging_organisation_id: merge_request_params.fetch(:requesting_organisation_id) })
```

Comment:
I'd say wrap this in a transaction block so if one create fails it'll rollback the other